boto3/examples/s3.rst

```diff
@@ -0,0 +1,15 @@
+**To list objects in an Amazon S3 bucket**
```

Comment:
Maybe so. It would likely just be normal headers (e.g., underlining text with a rst heading), and then a localtoc could potentially be injected during the build. I'll take a look.